public/app/features/explore/Logs.tsx

```diff
@@ -358,7 +369,7 @@ export default class Logs extends PureComponent<LogsProps, LogsState> {
           {hasData && deferLogs && <span>Rendering {dedupedData.rows.length} rows...</span>}
         </div>
         {!loading &&
-          !hasData &&
+          queryEmpty &&
```

Comment:
Something is not right with this logic (probably my fault for suggesting it). Now the Scan for older is no longer shown after you stop scanning, but it should be shown again.